cmd/maya-exporter/app/collector/collector.go

```diff
@@ -203,122 +205,41 @@ func (e *VolumeExporter) Collect(ch chan<- prometheus.Metric) {
 	avgWriteBlockCountPS.Collect(ch)
 	sizeOfVolume.Collect(ch)
 	volumeUpTime.Collect(ch)
+	connectionRetryCounter.Collect(ch)
```

Comment:
How about having these as properties of volume exporter struct. Can we avoid global variables?

Reply:
I also believe that all the fields should be tied with the structure but i want to avoid the repetition of the same into two structures.